gpii/node_modules/flowManager/test/configs/gpii.flowManager.tests.capture.fakeData.config.json5

```diff
@@ -11,6 +12,17 @@
                 "target": "{that flowManager solutionsRegistryDataSource}.options.path",
                 "priority": "after:flowManager.solutions"
             },
+            "capture.loadRepositorySolutions": {
```

Comment:
The same comment above applies here too.

Reply:
The distribution of a mock load-from-repository function isn't needed here in any absolute sense; however, in this case, the load registries sequence is in fact done properly, but as a mockup and it doesn't fail nor log any extra error messages.  The capture tests thereby show that the local SRDS works as described.  I thought this was relevant since the capture tool would be using the local SRDS in reality.  However, if you think this is overkill, I'll remove it.